app/src/main/java/de/westnordost/streetcomplete/data/visiblequests/QuestTypeOrderController.kt

```diff
@@ -27,6 +27,11 @@ class QuestTypeOrderController(
         })
     }
 
+    fun copyOrders(presetId: Long, newPresetId: Long) {
+        val orders = getOrders(presetId)
+        setOrders(orders, newPresetId)
```

Comment:
```suggestion
        questTypeOrderDao.setAll(newPresetId, questTypeOrderDao.getAll(presetId))
        if (newPresetId == selectedPresetId) onQuestTypeOrderChanged()
```

I think I prefer this because it does not take the detour of getting the quest type classes.